### PR TITLE
Document ambiguous Version field for ServiceUpdate

### PR DESCRIPTION
**Pull request by harrisonturton**
Currently, the behaviour for the version field in ServiceUpdate()
is vague. Without an correct version number, users are unable to
successfully run ServiceUpdate(), which is a pretty critical method
for scaling services (for example).

**- What I did**

Added extra documentation on the `client.ServiceUpdate()` method detailing what the `version` field does, why it's there, and how to find it.

**- Description for the changelog**
Update documentation for version field in `client.ServiceUpdate`

Cute duck

As referenced in [#38047](https://github.com/moby/moby/issues/38047#issuecomment-431784910), [#30794](https://github.com/moby/moby/issues/30794#issuecomment-429741548) and fix [#38040](https://github.com/moby/moby/pull/38040)

### CODE REVIEW

**Comment by harrisonturton** [comment]
Not sure how I requested code reviews from specific people – is that automatic? Still new to this!

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Comment by cpuguy83** [comment]
It's because dnephin is listed as a code owner for this part of the code base, GH automatically did it.

**Review by yongtang — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
SGTM, thanks!

ping @aaronlehmann ptal

**Comment by thaJeztah** [comment]
oh; we already have a lot of LGTM's - let's merge!

**Comment by thaJeztah** [comment]
Congrats with your first contribution merged 🎉

**Comment by harrisonturton** [comment]
Thanks everyone!

**Pull request merged**
